src/karma.config.js

```diff
@@ -0,0 +1,126 @@
+/* global process */
+
+let chromeFlags = [];
+
+process.env.CHROME_BIN = require('puppeteer').executablePath();
+
+// In Docker, the tests run as root, so the sandbox must be disabled.
+if (process.env.RUNNING_IN_DOCKER) {
```

Comment:
The `RUNNING_IN_DOCKER` section is only needed in Jenkins IIRC. Since we're using GitHub Actions I think we can drop this.